### PR TITLE
feat: add gitlab and github support for repositories under 4-appfactory

### PR DESCRIPTION
**Pull request by caetano-colin**

### CODE REVIEW

File: 4-appfactory/modules/app-group-baseline/main.tf [hunk 1]
```diff
@@ -37,10 +37,30 @@ locals {
...
+  org_ids           = distinct([for env in var.envs : env.org_id])
+  use_csr           = var.cloudbuildv2_repository_config.repo_type == "CSR"
+  service_repo_name = var.cloudbuildv2_repository_config.repositories[var.service_name].repository_name
 }
 
 
+module "cloudbuild_repositories" {
+  count = local.use_csr ? 0 : 1
+  # TODO: update git URL to registry format after release is available
+  source = "git::https://github.com/terraform-google-modules/terraform-google-bootstrap.git//modules/cloudbuild_repo_connection?ref=a6072e0db92d976f1535dab767ad6b4331bcb4ef"
```
Comment (gtsorbo) [inline 4-appfactory/modules/app-group-baseline/main.tf:49]: Can we update with registry release now? https://github.com/terraform-google-modules/terraform-google-bootstrap/releases/tag/v10.0.0

**Review by gtsorbo — CHANGES_REQUESTED** [review]

File: 4-appfactory/README.md [hunk 2]
```diff
@@ -31,6 +31,106 @@ It will also create an Application Folder to group your admin projects under it,
 
 ## Usage
 
+#### Cloud Build with Github Pre-requisites
+
+To proceed with github as your git provider you will need:
```
Comment (apeabody) [inline 4-appfactory/README.md:36]:
```suggestion
To proceed with GitHub as your git provider you will need:
```

File: 4-appfactory/README.md [hunk 3]
```diff
@@ -31,6 +31,106 @@ It will also create an Application Folder to group your admin projects under it,
...
+         }
+      }
+      github_secret_id                            = "projects/REPLACE_WITH_PRJ_NUMBER/secrets/github-pat" # Personal Access Token Secret
+      github_app_id_secret_id                     = "projects/REPLACE_WITH_PRJ_NUMBER/secrets/github-app-id" # App ID value secret
+   }
+   ```
+
+#### Cloud Build with Gitlab Pre-requisites
+
+To proceed with gitlab as your git provider you will need:
```
Comment (apeabody) [inline 4-appfactory/README.md:99]:
```suggestion
To proceed with Gitlab as your git provider you will need:
```

File: test/integration/appfactory/appfactory_test.go [hunk 4]
```diff
@@ -174,6 +174,8 @@ func TestAppfactory(t *testing.T) {
 					}
 					// triggers
 					repoName := appData.Get("app_infra_repository_name").String()
+					triggerRegion := "us-central1"
```
[the remark this answers is not in the document]
Comment (apeabody) [reply test/integration/appfactory/appfactory_test.go]: Thanks @caetano-colin! If we'd done that originally, wouldn't have had to change it now.

**Pull request merged**